src/packages/core/property-type/workspace/manifests.ts

```diff
@@ -43,7 +43,7 @@ export const manifests: Array<UmbExtensionManifest> = [
 		},
 		conditions: [
 			{
-				alias: 'Umb.Condition.WorkspaceAlias',
+				alias: UMB_WORKSPACE_ALIAS_CONDITION,
```

Comment:
@warrenbuckley should it be renamed to `UMB_WORKSPACE_CONDITION_ALIAS`? It would align with `UMB_PROPERTY_TYPE_WORKSPACE_ALIAS` and `UMB_DOCUMENT_TYPE_WORKSPACE_ALIAS`.

Reply:
They align with the others found in `src/packages/core/workspace/conditions/const.ts`

Reply:
There is no other alias specific constant in that file?

Only these:

```
export const UMB_WORKSPACE_HAS_COLLECTION_CONDITION = 'Umb.Condition.WorkspaceHasCollection';
export const UMB_WORKSPACE_ENTITY_IS_NEW_CONDITION = 'Umb.Condition.WorkspaceEntityIsNew';
```

It seems to be a name convention using `_ALIAS` as suffix:
https://github.com/search?q=repo%3Aumbraco%2FUmbraco.CMS.Backoffice+constants.ts&type=code

Reply:
`UMB_WORKSPACE_HAS_COLLECTION_CONDITION` used here:
https://github.com/umbraco/Umbraco.CMS.Backoffice/blob/d8b81ac7861ef1c3353c78f9cea1e71da69522ca/src/packages/core/workspace/conditions/workspace-has-collection.condition.ts#L32

`UMB_WORKSPACE_ENTITY_IS_NEW_CONDITION` used here:
https://github.com/umbraco/Umbraco.CMS.Backoffice/blob/d8b81ac7861ef1c3353c78f9cea1e71da69522ca/src/packages/core/workspace/conditions/workspace-entity-is-new.condition.ts#L3

Reply:
I must agree with @bjarnef here. It should have `_ALIAS` as a suffix. You could re-export the others as such as well and deprecate the old ones, @warrenbuckley?

Reply:
Can you show me an example of how to do a deprecation and will update later tonight after work @iOvergaard

Reply:
Sure thing, you add a jsdoc on top of the variable:

```js

/**
 * This is my workspace condition alias
 */
export const UMB_VARIABLE_ALIAS = 'variable';

/**
 * This is my workspace condition alias
 * @deprecated Use WHATEVER instead. This will be removed in Umbraco 16.
 */
export const UMB_VARIABLE = UMB_VARIABLE_ALIAS;